sqlite3.go

```diff
@@ -253,6 +255,17 @@ type SQLiteRows struct {
 	done     chan struct{}
 }
 
+// SQLitePreUpdateData represents all of the data available during a
+// pre-update hook call.
+type SQLitePreUpdateData struct {
```

Comment:
Please move this to a file named sqlite3_opt_preupdate_hook.go and set a build tag sqlite_preupdate_hook